src/registrar/assets/sass/_theme/_uswds-theme-custom-styles.scss

```diff
@@ -133,11 +133,20 @@ a.breadcrumb__back {
   }
 }
 
-.withdraw_outline {
+a.withdraw_outline {
   box-shadow: inset 0 0 0 2px color('error');
   color: color('error');
 }
 
+a.withdraw_outline:visited {
+  box-shadow: inset 0 0 0 2px color('error');
+  color: color('error');
+}
+
+a.withdraw {
+  background-color: color('error');
+}
+
```

Comment:
We need to add a couple of more rules for the `hover`, `focus`, and `active` states, because otherwise it's using the `usa-button` versions, which are the wrong hue. This was underspecified on my part, sorry about that.

On `.withdraw_outline`, could you add for `:hover` and `:focus` :
`color: color('error-dark');`

and for `:active`:
`color: color('error-darker');`

Similarly  for `withdraw` for or `:hover` and `:focus` :
`background-color: color('error-dark');`

and for `:active:`
`background-color: color('error-darker');`